### DISCUSSION TITLE
Clarification about safety for DLX publish in case of single node

### DISCUSSION THREAD

**Discussion by Guuzen** [rabbitmq/rabbitmq-website#1258] [General]
Does it mean that if there is only single node then using DLX is safe?
https://github.com/rabbitmq/rabbitmq-website/blob/8b3bfb1aa3f81f997726c969997968483cea8222/site/dlx.md#safety

**Comment by michaelklishin** [comment]
I will convert this issue to a GitHub discussion. Currently GitHub will [automatically close and lock the issue](https://github.com/github/feedback/discussions/3197) even though your question will be transferred and responded to elsewhere. This is to let you know that we do not intend to ignore this but this is how the current GitHub conversion mechanism makes it seem for the users :(

**Comment by michaelklishin** [comment]
The root cause is that dead lettering does not use publisher confirms. The number of nodes is irrelevant. In practice dead lettering is reasonably safe to be practical with any number of nodes.
